ui/selectors/selectors.js

```diff
@@ -171,6 +171,21 @@ export const getMetaMaskAccounts = createSelector(
     ),
 );
 
+export function getCompletedOnboarding(state) {
+  return state.metamask.completedOnboarding;
```

Comment:
In keeping with the "ducks" pattern, we've started placing selectors alongside the reducers they correspond to. So these would belong at the end of `ui/ducks/metamask/metamask.js`.